src/commands/deploy.js

```diff
@@ -138,13 +138,24 @@ exports.handler = async (args = {}) => {
     const formattedServices = formatServices(res.deployments);
     formattedServices.forEach(({name, domain, host}) => {
       resultTable.push([name, domain, host]);
-      if(args.open){
-        open('http://' + domain);
-      }
     });
 
     // draw table
     console.log(resultTable.toString());
+
+    // open in browser
+    if(args.open) {
+      // check if domain is set
+      if(formattedServices[0].domain != 'not set' && typeof(formattedServices[0].domain) != 'undefined') {
+        // check if multiple domains
+        if(formattedServices[0].domain.includes(",")){
```

Comment:
I think you can just always use `formattedServices[0].domain.split(',')[0].trim()` since even if there's no commas - it'll be just a single value array. This will remove that additional `if/else` here.